.travis.yml

```diff
@@ -31,6 +31,7 @@ env:
       - EXTRA_CONTEXT='include_example_cython_code=y'
       - EXTRA_CONTEXT="package_name=AstropyProject" FOLDERNAME='AstropyProject'
       - EXTRA_CONTEXT='_parent_project=sunpy'
+      - EXTRA_CONTEXT='__minimum_python_version__=3.5'
```

Comment:
@Cadair , you mean like this? I can't say I am familiar with template testing.

Reply:
yes I think so.